docs/source/noise/index.rst

```diff
@@ -114,51 +114,51 @@ Noise types for each column
     - Types of noise
     - Notes
   * - First name
-    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
-    - Leave a field blank, use a fake name, use a nickname, make typos, make OCR errors, make phonetic errors
-    -
+    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, 1040, SSA
+    - Leave a field blank, use a fake name, nickname, make typos, make OCR errors, make phonetic errors
+    - In the 1040 form, the same noise types apply to the first name columns for the joint filer and dependents
   * - Middle initial
-    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
+    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, 1040, SSA
     - Leave a field blank, make typos, make OCR errors, make phonetic errors
-    -
+    - In the 1040 form, the same noise types apply to the middle initial columns for the joint filer and dependents
   * - Last name
-    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
+    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, 1040, SSA
     - Leave a field blank, use a fake name, make typos, make OCR errors, make phonetic errors
-    -
+    - In the 1040 form, the same noise types apply to the last name columns for the joint filer and dependents
   * - Age
-    - Decennial Census, ACS, CPS, W-2 and 1099
+    - Decennial Census, ACS, CPS
```

Comment:
```suggestion
    - Decennial Census, ACS, CPS, W-2 and 1099
```
Removing age column from W2/1099 [hasn't been implemented yet](https://jira.ihme.washington.edu/browse/MIC-4134), and I don't know whether the change will make it into yellow release, so let's not make this change in the docs yet.